lib/sidekiq-monitoring.rb

```diff
@@ -2,7 +2,7 @@
 require 'multi_json'
 
 class SidekiqMonitoring < Sinatra::Base
-  VERSION = "1.2.0"
+  VERSION = "1.2.1"
```

Comment:
I would jump to 1.3.0 as there are a lot of changes and breaking changes in configuration

Reply:
You're right!